deploy/addon/pulsar.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Pulsar
+shortdesc: A distributed messaging and streaming platform on the publish/subsribe model
+tags:
+- addons
+keywords:
+- pulsar
+- messaging
+- cold storage
+- queue
+- hot storage
+- biscuit
+---
+
+## Overview
+
+Pulsar works on a publisher / subscriber model, just like RabbitMQ or Kafka.
+
+```
+client               Pulsar             client
+publishes to   =>    topic      <=  subscribes to
+
+"producer"                           "consumer"
+```
+
+They are several modes of subscription. A consumer may subscribe exclusively, or share the subscription with other consumers.
+
+- Exclusive
```

Comment:
What is this list ?

Reply:
subscription type list, I added clear title.